cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -156,6 +156,7 @@ instance HasSeverityAnnotation (TraceChainSyncClientEvent blk) where
   getSeverityAnnotation (TraceFoundIntersection _ _ _) = Info
   getSeverityAnnotation (TraceRolledBack _) = Notice
   getSeverityAnnotation (TraceException _) = Warning
+  getSeverityAnnotation (TraceTermination _) = Warning
```

Comment:
I don't think other mini-protocols use `Warning` for tracing termination.  Terminating min-protocols will be a normal mode of operation, so `Warning` seems to high severity for me.  Could you check what other mini-protocols are using?